content/spin/v2/spin-in-pods-legacy.md

```diff
@@ -338,67 +335,70 @@ The current Spin k8s plugin relies on Docker and Kubectl under the hood. It’s
 
 ### Install Plugin
 
-To install this plugin, simply run:
+To install this plugin, run:
 
-```yaml
+<!-- @selectiveCpy -->
+
+```console
 spin plugin install -u https://raw.githubusercontent.com/chrismatteson/spin-plugin-k8s/main/k8s.json
```

Comment:
```suggestion
$ spin plugins update
$ spin plugins install k8s
```

Reply:
The subsequent paragraphs still talk about the Chris Matteson plugin way of doing things: we should not direct them to the new plugin on the page that gives instructions for the old one.

(Unless we are adding the old plugin to the registry, which seems like a bad idea right now.)

Reply:
Keeping the old plugin way of doing things on the old instructions ✔️